Patches/Vanilla Factions Expanded - Settlers/ThingDefs_Buildings/TurretGatlingGun.xml

```diff
@@ -0,0 +1,124 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+
+  <Operation Class="PatchOperationFindMod">
+    <mods>
+      <li>Vanilla Factions Expanded - Settlers</li>
+    </mods>
+
+    <match Class="PatchOperationSequence">
+      <operations>
+        <!-- === Gun === -->
+        <li Class="CombatExtended.PatchOperationMakeGunCECompatible">
+          <defName>Gun_GatlingGun</defName>
+
+          <statBases>
+            <Bulk>15.54</Bulk>
+            <ShotSpread>0.04</ShotSpread>
+            <SwayFactor>1.16</SwayFactor>
+            <SightsEfficiency>1</SightsEfficiency>
+            <RangedWeapon_Cooldown>0.36</RangedWeapon_Cooldown>
+          </statBases>
+
+          <Properties>
+            <recoilAmount>1.52</recoilAmount>
+            <verbClass>CombatExtended.Verb_ShootCE</verbClass>
+            <hasStandardCommand>True</hasStandardCommand>
+            <defaultProjectile>Bullet_4570Gov_FMJ</defaultProjectile>
+            <burstShotCount>10</burstShotCount>
+            <ticksBetweenBurstShots>9</ticksBetweenBurstShots>
+            <warmupTime>1.4</warmupTime>
+            <range>62</range>
+            <soundCast>VFE_Shot_GattlingGun</soundCast>
+            <soundCastTail>GunTail_Medium</soundCastTail>
+            <muzzleFlashScale>10</muzzleFlashScale>
+            <recoilPattern>Mounted</recoilPattern>
+          </Properties>
+
+          <AmmoUser>
+            <magazineSize>40</magazineSize>
+            <reloadTime>7.8</reloadTime>
+            <ammoSet>AmmoSet_4570Gov</ammoSet>
+          </AmmoUser>
+
+          <FireModes>
+            <aiUseBurstMode>false</aiUseBurstMode>
+            <aimedBurstShotCount>5</aimedBurstShotCount>
+            <aiAimMode>SuppressFire</aiAimMode>
+          </FireModes>
+        </li>
+
+        <!-- === Turret === -->
+        <!-- == thingClass == -->
+        <li Class="PatchOperationReplace">
+          <xpath>/Defs/ThingDef[defName="Turret_GatlingGun"]/thingClass</xpath>
+
+          <value>
+            <thingClass>CombatExtended.Building_TurretGunCE</thingClass>
```

Comment:
Not adding ammo to it (I might have missed that)

Reply:
Dunno why this is missing, they of course have ammo as i've been using them for almost a year in game. Something must've messed up because the branch is indeed up to date with my local changes.